tests/FillingTheDishwasher.tex

```diff
@@ -0,0 +1,67 @@
+\section{Filling the Dishwasher [DSPL \& OPL]}
+The robot helps by placing all dirty dishes inside the dishwasher
+
+\subsection{Goal}
+The robot has to correctly identify and manipulate cutlery, kitchenware, and tableware, taking them from the sink and safely placing them in the dishwasher rack.
+
+\subsection{Focus}
+This test focuses on object recognition and manipulation, paying particular attention in orientating objects for an adequate placement in narrow spaces.
+
+\subsection{Setup}
+\begin{enumerate}
+	\item \textbf{Dishwasher:} The dishwasher has the door open and its bottom rack half way outside, ready to be filled in.
+	\item \textbf{Location:} This test takes place in the kitchen of the arena. 
+	\item \textbf{Sink:} The sink has 10 utensils (See \ref{rule:tableware_kitchenware}) in it. There are at least two dishes (or bowls) two mugs (or glasses, or cups), one cooking pan (or pot), and one spoon (or any other cutlery).
+	\item \textbf{Start position:} The robot will start somewhere between the \textit{sink} and the \textit{dishwasher}.
+\end{enumerate}
+
+
+\subsection{Task}
+The robot must execute the following tasks in any order as long as there are utensils in the sink.
+\begin{itemize}
+	\item \textbf{Place cutlery:} The robot takes any cutlery from the sink and safely places it in the cutlery basket of the dishwasher.
```

Comment:
I'm fine with these being on a table. The whole point is to not have to bring the dishes anywhere, the robot does this all for you. 
To speed things up, we can put the table close to the dishwasher (they happen to be < 1.5m apart in my home)

Reply:
The *sink* is like any other location. We can define a table to be the sink. If you check the General Rules, we are not asking for a real one so far. However, assuming we have one, I was thinking in something like this.

| Side by side  | Around the corner |
| ------------- | ------------- |
| ![fd0b781a869f6b643c0e916196fbb449](https://user-images.githubusercontent.com/8583949/34051216-fdf4e9f4-e1bd-11e7-944e-14a1bfc1c39c.jpg)  | 
![img_3225](https://user-images.githubusercontent.com/8583949/34051229-09525854-e1be-11e7-97c8-d072d435207b.jpg)  |

I don't want to overspecify the rulebook